### ISSUE TITLE
Make togglws reconnect to the server automatically after ~50 mins

### ISSUE THREAD

**Issue by Brads3290** [Brads3290/toggl_webhooks#6]
See my quote from issue #5 :
>Note that this reconnects at 1 hour exactly, after the server closes the connection. In my test, the Toggl web app reconnected at around 58 mins which suggests that Toggl places the responsibility on the client to close the connection and reconnect before the hour, so maybe we should implement additional logic to do that.

**Comment by ihoru** [comment]
Why did you open this issue? :)

**Comment by Brads3290** [comment]
Sort of a to-do for myself in future. It looks like Toggl makes the assumption that the client will reconnect on a schedule (rather than only when Toggl server closes the connection), so there could be an additional risk that Toggl with make a change on their server that breaks our code.

Of course that's always a risk but I figure the more closely we mimic their web app, the less risk there is.

**Comment by ihoru** [comment]
In this case I didn't understand what have you done in #5

**Comment by Brads3290** [comment]
In #5 I made it wait for the connection to be closed by the server, then automatically reconnect.

I made this issue because I want to add an extra layer of “protection” by proactively recycling the connection rather than waiting for the server to close it, because that’s how the Toggl web app works.

Hopefully that makes more sense :)

**Comment by ihoru** [comment]
Yes, I understood now. With those changes (#5) I still got disconnected (at least once, I don't want to check any more, I just restart app every 30 minutes).
